src/components/List/styles.ts

```diff
@@ -0,0 +1,51 @@
+import theme from '../../theme/theme';
+import {Text, View} from 'react-native';
+import {Platform} from 'react-native';
+import styled from 'styled-components';
+
+export const CryptoContainer = styled(View)`
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0px 24px;
+  padding: 30px 0px;
+  border-bottom-width: 1px;
+  border-color: ${theme.colors.gray};
```

Comment:
Is optional, but you can use the ThemeProvider of styled components to avoid importing the theme in every style file.

Reply:
You're right, it is more future prove that way